web/src/components/organisms/Project/ModelsMenu/hooks.ts

```diff
@@ -9,7 +10,7 @@ import {
   Model as GQLModel,
 } from "@reearth-cms/gql/graphql-client-api";
 import { useT } from "@reearth-cms/i18n";
-import { useModel } from "@reearth-cms/state";
+import { useModel, useWorkspace } from "@reearth-cms/state";
```

Comment:
Can we use useProject here as well and remove using usParams for projectID? 
Here we only want to get the modelId from the route and rely on the header managing the current workspace and project

Reply:
@KaWaite , also updated.
https://github.com/reearth/reearth-cms/blob/a92e02ae442c0515eece917d5004c0ad94bd662c/web/src/components/organisms/Project/ModelsMenu/hooks.ts#L13